### PR TITLE
Added Ctrl+Shift+C to copy path

### LINKED ISSUE #6213: Add ctrl + shift + c to copy item path
### What's the Problem?

_No response_

### Solution/Idea

While Files already supports copying an items path, it requires users to open the context menu. Adding a shortcut will make this feature more accessible.

### Alternatives

NA

### Priorities

_No response_

### Files Version

Version: 2.0.13.0

### Windows Version

OS Version: 10.0.22463.1000

### Additional comment

_No response_

### PR DESCRIPTION
**Pull request by d2dyno1**
**Resolved / Related Issues**
- Closes #6213

**Details of Changes**
- Ctrl+Shift+C will now copy the path at directory

**Validation**
How did you test these changes?
- [x] Built the app

**Screenshots (optional)**
Add screenshots here.

### CODE REVIEW

**Comment by gave92** [comment]
Is this also supposed to allow copying the path of the selected item?

File: Files/Views/ColumnShellPage.xaml [hunk 1]
```diff
@@ -30,6 +30,11 @@
             Invoked="KeyboardAccelerator_Invoked"
             IsEnabled="{x:Bind IsCurrentInstance, Mode=OneWay}"
             Modifiers="Control" />
+        <KeyboardAccelerator
```
Comment (yaichenbaum) [inline Files/Views/ColumnShellPage.xaml:33]: With the code behind detecting key presses, is this actually doing anything?

Comment (d2dyno1) [reply Files/Views/ColumnShellPage.xaml:33]: Yes this is necessary otherwise the function won't be invoked.

**Review by yaichenbaum — APPROVED** [review]

File: Files/Views/ModernShellPage.xaml.cs [hunk 2]
```diff
@@ -638,6 +639,29 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
...
                     break;
 
+                case (true, true, false, true, VirtualKey.C):
+                    {
+                        DataPackage dataPackage = null;
+
+                        if (!InstanceViewModel.IsPageTypeSearchResults && ContentPage.SelectedItem != null)
+                        {
+                            dataPackage = new DataPackage();
+                            dataPackage.SetText(this.FilesystemViewModel.WorkingDirectory);
```
Comment (gave92) [inline Files/Views/ModernShellPage.xaml.cs:649]: ContentPage.SelectedItem.ItemPath?

File: Files/Views/ModernShellPage.xaml.cs [hunk 3]
```diff
@@ -638,6 +639,29 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
...
+                        }
+                        else
+                        {
+                            dataPackage = new DataPackage();
+                            dataPackage.SetText(ContentPage.SelectedItem.ItemPath);
```
Comment (gave92) [inline Files/Views/ModernShellPage.xaml.cs:654]: this.FilesystemViewModel.WorkingDirectory?

File: Files/Views/ColumnShellPage.xaml.cs [hunk 4]
```diff
@@ -598,6 +599,29 @@ private async void KeyboardAccelerator_Invoked(KeyboardAccelerator sender, Keybo
...
+
+                        if (!InstanceViewModel.IsPageTypeSearchResults && ContentPage.SelectedItem != null)
+                        {
+                            dataPackage = new DataPackage();
+                            dataPackage.SetText(this.FilesystemViewModel.WorkingDirectory);
+                        }
+                        else
+                        {
+                            dataPackage = new DataPackage();
+                            dataPackage.SetText(ContentPage.SelectedItem.ItemPath);
```
Comment (gave92) [inline Files/Views/ColumnShellPage.xaml.cs:614]: this.FilesystemViewModel.WorkingDirectory?

**Comment by d2dyno1** [comment]
:eyes: Thank you @gave92! I really need to get my eyes checked...

**Comment by gave92** [comment]
You know? Perhaps you can replace everything with `SlimContentPage?.CommandsViewModel.CopyPathOfSelectedItemCommand.Execute(null)`.

**Review by gave92 — APPROVED** [review]

**Comment by yaichenbaum** [comment]
@d2dyno1 thank you!

**Pull request merged**